chevah_build

```diff
@@ -258,6 +292,18 @@ case $OS in
         # "10.4", and then tries to avoid the broken readline in OS X 10.4.
         export MACOSX_DEPLOYMENT_TARGET=10.12
         ;;
+
+    openbsd*)
+        export CC=clang
+        export CXX=clang++
+        # Fix from upstream, newer cryptography versions should work as-is.
+        PIP_LIBRARIES="\
+            setproctitle==1.1.10 \
+            cryptography==2.2.2chevah2 \
```

Comment:
I don't understand these PIP_LIBRARIES set.
For example here we have 2.2.2chevah2 and on other places we have 2.2.2chevah3

I would to have something like a LATEST_LIBRARIES variable defined and documented at the top of the script, and then here only have `PIP_LIBRARIES=LATEST_LIBRARIES`

Reply:
There are multiple fixes in the 2.2.2chevah versions. I tried to document them in the comments for every occurrence. Eg., for 2.2.2chevah2, you can read in the comment above that it includes a fix from upstream that will be included in 2.2.3. 

Obviously, this is a fix specific for this OS: OpenBSD. Cryptography 2.2.2 (and I think any older version too) will not work on OpenBSD 6.3 because of an incompatibility with the LibreSSL libs bundled with OpenBSD 6.3. 

So it's not possible to use `LATEST_LIBRARIES` **unless** you are willing to use these fixes for all OS'es. I have chosen to use pristine upstream sources wherever possible (so everywhere except OpenBSD, HP-UX and Solaris 11).

Reply:
is hard to follow the commit in here.
I would like to see the changes made in a brank in our fork of cryptography and have those versions in pypi.chevah.com 
https://github.com/chevah/cryptography/branches